test/main_test.go

```diff
@@ -49,19 +50,23 @@ func TestMain(m *testing.M) {
 
 type queryEntry struct {
 	dir      string
-	kind     model.FileKind
+	kind     []model.FileKind
 	platform string
 }
 
 func (q queryEntry) getSampleFiles(tb testing.TB, filePattern string) []string {
-	files, err := filepath.Glob(path.Join(q.dir, fmt.Sprintf(filePattern, strings.ToLower(string(q.kind)))))
-	x0 := filepath.FromSlash(path.Join(q.dir, "test/positive_expected_result.json"))
-	for i, check := range files {
-		if check == x0 {
-			files = append(files[:i], files[i+1:]...)
+	var files []string
+	for _, kinds := range q.kind {
+		kindFiles, err := filepath.Glob(path.Join(q.dir, fmt.Sprintf(filePattern, strings.ToLower(string(kinds)))))
+		x0 := filepath.FromSlash(path.Join(q.dir, "test/positive_expected_result.json"))
+		for i, check := range kindFiles {
```

Comment:
this is all just for skipping `test/positive_expected_result.json` right?

Reply:
yes